ext/SolidStateDetectorsGeant4Ext.jl

```diff
@@ -220,9 +220,12 @@ function SolidStateDetectors.run_geant4_simulation(app::G4JLApplication, number_
         evtno += 1
     end
 
-    evts_with_points = map(evt -> merge(evt, (pos = cartesian_zero + evt.pos,)), evts)
+    evts_with_points = map(evts) do evt
+        merge(evt, (pos = SolidStateDetectors.to_internal_point(evt.pos),))
+    end
```

Comment:
The tests (Documentation) are failing because it expects `evts.pos` to have units of `m` that are stripped in the documentation here:
https://github.com/JuliaPhysics/SolidStateDetectors.jl/blob/a8518b65f08cfcf78c49091a1140d3e6995c1c97/docs/src/tutorials/geant4_ssd_lit.jl#L92